core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceTypeUtil.java

```diff
@@ -50,6 +51,7 @@
                     .put(Double.class, Double::valueOf)
                     .put(String.class, Function.identity())
                     .put(UUID.class, UUID::fromString)
+                    .put(Period.class, Period::parse)
```

Comment:
Could you update the documentation?
https://github.com/line/armeria/blob/2f270812aab77dd436a74b6a8d8223f0f05fc65c/site/src/pages/docs/server-annotated-service.mdx#L177-L186

Reply:
OMG, thanks!